madgraph5.sh

```diff
@@ -0,0 +1,38 @@
+package: madgraph5
+version: "%(tag_basename)s%(defaults_upper)s"
+source: https://github.com/shir994/madgraph5
```

Comment:
Once this works this should be hosted under `ShipSoft`
@ThomasRuf

Reply:
True. For now its just a dummy repository to make aliBuild work.